inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -591,7 +609,7 @@ protected function get_url( $post ) {
 		 *
 		 * @see https://wordpress.org/plugins/page-links-to/ can rewrite permalinks to external URLs.
 		 */
-		if ( false === strpos( $url['loc'], $this->get_home_url() ) ) {
+		if ( false === strpos( $url['loc'], $this->get_home_proto_rel_url() ) ) {
```

Comment:
If you use the classifier, you'll have to replace the content of the `if` with `$this->classifier->classify( $url['loc'] === WPSEO_Link::TYPE_EXTERNAL )`.